src/handbook/design/branding.md

```diff
@@ -1,6 +1,120 @@
 # FlowFuse Branding Guidelines
 
 For FlowFuse Assets (e.g. logos, pictograms, and raw design files), please look in the [`/design/Media`](https://drive.google.com/drive/u/1/folders/1guBnBUrIiRXuK6vsik_NIXIhtE1cZRRa) folder on Google Drive.
+
+## Logo
+### Composition
+
+Our logo is a combination mark, featuring an abstract symbol that represents the flows connecting the nodes, along with a wordmark displaying our name.
+
+### Versions
+
+**The horizontal version is the preferred choice**. When space and layout constraints make this option an awkward fit, forcing the logo to be too small, the vertical version can be used.
+
+#### Horizontal Version
+
+<hr style="margin-bottom:30px; border-color:#E5E7EB;">
+<div class="flex h-full flex-col justify-center items-center md:mt-0 mb-14">
+    <img width="250" min-width="250" alt="Image of the horizontal version of FlowFuse logo" src="../images/logos/ff-logo--wordmark--light.png">
+</div>
+
+#### Vertical Version
+
+<hr style="margin-bottom:30px; border-color:#E5E7EB;">
+<div class="flex h-full flex-col justify-center items-center md:mt-0 mb-14">
+    <img width="180" min-width="180" alt="Image of the vertical version of FlowFuse logo" src="../images/logos/ff-logo--square--light.png">
+</div>
+
+### Minimun Sizes
```

Comment:
```suggestion
### Minimum Sizes
```

Reply:
Is there a difference?

Reply:
`Minimun` (with an 'n' at the end) isn't a word :)

Reply:
Ohhh I see now!! :ok_hand: :ok_hand: